Missions/LOG_Suministros.sqf

```diff
@@ -213,11 +218,14 @@ else
 			};
 	};
 
-_camion setFuel 0;
```

Comment:
Here i make the crate become empty after the supplies are successfully delivered